database/comments.js

```diff
@@ -0,0 +1,40 @@
+const comments = [
+  'Yo Please Check My Beats Out! Let me know how they are!',
+  'im a new producer follow me and hmu for a prod',
+  'yo check out my new song if yall want!! i just released!! if yall mess with give me a follow and a like and help me get to 300 followers!!Thx and peace!',
+  'hi',
+  'I WOULD REALLY APPRECIATE IF YOU COULD CHECK OUT MY LATEST SONG AND LET ME KNOW WHAT YOU THINK',
+  'Check out my ep',
+  'go check out my new song Im Just Sayin....',
+  'bruh, its like his music got soft when he came up',
+  'yall plz check out my recent, ill like or follow back if you follow me',
+  'Hey, whats up? If you are an artist, producer or just love to listen a good music, you should check out my beats! Im sure you are going to like them! Maybe, I will INSPIRE you to CREATE something GREAT. Ill appreciate if you will follow me and leave in comments your feelings about my beats. Feel free to HMU if u have any questions! Thank you and Good LUCK. Bro!',
+  'yahhhhhh',
+  'Ayeeeee',
+  'fireeee',
+  'KANYE TOOK MY FLOW !!! PROOF ON MY PAGE RACKS TUPAC TOO',
+  'Go CHECK OUT my music it’s lit as fire better check it out before I blow up',
+  'Check out my profile. Just released two new songs. I follow back too',
+  'please listen to my song i beg u',
+  'go bump my music before im famous',
+  'this song still gooooeesss',
+  'broke my heart oh no you didnt',
+  'peep my fire',
+  'i make sad/chill hiphop music.. wud highly appreciate u checking it out <3',
+  'stright fire',
+  'I swear when he says i got water in my red cup i get lit',
+  'no one cares about your weak music dang yall find the first kid yall see blow up and act like they Rey just give you clout',
+  'Sounds lit, not very good lyrics tho.',
+  'This song is a straight banger',
+  'Ahh I am making it to the top',
+  'Lowkey noice',
+  'yaya dis the new wave',
+  'no',
+  'WEAAAAAK',
+  'yo this wak',
+  'lol i saw him when he first up',
+  'banggaaa fire',
+  'on REPEAT my ears bless up',
+];
+
+module.exports.comments = comments;
```

Comment:
Since you're only exporting one thing here, I think you can just say:

module.exports = comments.

Then, in line 11 of database/startup.js, you won't have to call comments.comments